account_bank_statement_import_camt_oca/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# © 2013-2017 Therp BV <http://therp.nl>
```

Comment:
© has to be replaced as per https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py